frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -99,45 +100,41 @@ const MyBotsPage = () => {
+            const connectorClass = isCompleted ? "step-connector-active" : "";
+
+            return (
+              <div key={s} className="flex flex-col items-center relative z-10">
+                <div className={`step-bubble ${bubbleClass}`}>
+                  {isCompleted ? <Check className="w-5 h-5" /> : s}
+                </div>
+                <span className={`mt-3 text-sm font-medium ${labelClass}`}>
+                  {labelText}
+                </span>
+                {idx < 2 && (
+                  <div className={`step-connector ${connectorClass}`}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)